protocol-designer/src/components/molecules/Checkpoint/Checkpoint.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box, COLORS, Flex, SPACING } from '@opentrons/components'
+
+import type { PropsWithChildren } from 'react'
+
+interface Props {
```

Comment:
```suggestion
interface CheckpointProps {
```
Would prefer this just for code consistency and not needing to cast imports when this interface is imported elsewhere.

Reply:
Hm, happy to do that, but, that's kind of surprising to me. You prefer that over `ComponentProps<Checkpoint>` (where `ComponentProps` comes from `React`)? Seems like it would pollute the global namespace less.

Reply:
I don't prefer it, but it's the pattern we follow in our codebase, and IMO consistency is more important on this one.